### ISSUE TITLE
Make this repo readonly?

### ISSUE THREAD

**Issue by sm4rk0** [cmprescott/ansible-xml#132]
From time to time, people are creating issues here while they should be filed at Ansible repo.
Is there any reason to leave this repo RW?

**Comment by cmprescott** [comment]
I agree. I am right in thinking I should use the "Archive this Repository" option? 

![image](https://user-images.githubusercontent.com/4502791/57875085-aca9e480-77d7-11e9-8478-d4cd29b2b484.png)

**Comment by timlnx** [comment]
Archived looks good so long as there is a huge plain and obvious redirect
message at the top of the readme.
